In [1]:
# Variance
ages = [23, 43, 22, 56, 74, 32, 68, 98, 45, 32]
print(ages)

[23, 43, 22, 56, 74, 32, 68, 98, 45, 32]


In [2]:
# mean value of ll the ages
import numpy as np
mean_ages = np.mean(ages)
print("Mean Value of all the ages is:", mean_ages)

Mean Value of all the ages is: 49.3


In [3]:
# Variance of all the ages
variance_ages = np.var(ages)
print("Variance of all the ages is:", variance_ages)

Variance of all the ages is: 547.01


# Observation:

As the Variance of the ages is very high, so the spread of the data is very high.

In [4]:
# Standard Deviation of all the ages
standard_ages = np.std(ages)
print("Standard eviation of all the ages is:", standard_ages)

Standard eviation of all the ages is: 23.388244910638335


# Observation:

As the standard deviation is 23.388, so it is 23.388 away from the mean.

In [6]:
import pandas as pd

data = [[10, 12, 13], [43, 23, 45], [32, 34, 21]]

# creating the dataframe
df = pd.DataFrame(data, columns = ["A","B","C"])

print(df)

    A   B   C
0  10  12  13
1  43  23  45
2  32  34  21


In [7]:
# To get the variance for the dataset columns wise
variance_ans = df.var(axis = 0)
print(variance_ans)

A    282.333333
B    121.000000
C    277.333333
dtype: float64


In [8]:
# To get the standard deviation for the dataset column wise

std_ans = df.std(axis = 0)
print(std_ans)

A    16.802778
B    11.000000
C    16.653328
dtype: float64
